src/drivers/imu/mpu9250/mpu9250.cpp

```diff
@@ -139,7 +139,11 @@ MPU9250::MPU9250(device::Device *interface, device::Device *mag_interface, const
 	_gyro_range_scale(0.0f),
 	_gyro_range_rad_s(0.0f),
 	_dlpf_freq(MPU9250_DEFAULT_ONCHIP_FILTER_FREQ),
+#ifdef CONFIG_ARCH_BOARD_CRAZYFLIE
```

Comment:
Can we find a way to do this without board specific defines in drivers? How about defaulting to 200 Hz for I2C?

Reply:
@dagar I can get it to run up to 330Hz on the crazyflie but at this frequency I get some weird behavior because the delay between two samples becomes quite variable. It seems to work well between 200 and 250Hz. Could changing this create issues on other architectures?

Reply:
Isn't the important difference SPI vs I2C rather than anything crazyflie specific?

Reply:
Ok I can fix this